src/management-system-v2/components/bpmn-viewer.tsx

```diff
@@ -37,19 +46,48 @@ const BPMNViewer = ({ definitionId, reduceLogo, fitOnResize }: BPMNViewerProps)
   // Allows for rerendering when the process changes but not the BPMN.
   const bpmn = useMemo(() => ({ bpmn: data }), [data]);
 
-  useEffect(() => {
-    console.log('viewer', viewer.current);
-    //viewer.current!.fitViewport();
-  }, []);
+  return (
+    <div
+      className={cn(style['bpmn-viewer-container'], {
+        [style['no-scroll']]: !scrollable,
+      })} /* Prevents the Viewer from beeing scrollable / zoomable */
+    >
```

Comment:
I think a better solution is to disable the `ZoomScroll` module of bpmn-js. Alternatively we can also use the `Viewer` instead of the `NavigatedViewer` in `BPMNCanvas` but then we would not be able to use navigation in other cases where we might want to.

Reply:
Opted for Adding the simple `Viewer` vs `NavigatedViewer` in the `BPMNCanvas` component. Instead of having types 'viewer' and 'modeler', there is also the 'navigatedviewer' as a third choice.